integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -130,8 +130,20 @@ spec:
           - configMapRef:
               name: {{ $fullName }}-config
 {{- $probePort := $isHaEmbedded | ternary "embedded" "rpc" }}
-{{ include "alluxio.master.readinessProbe" (dict "port" $probePort) | indent 10 }}
-{{ include "alluxio.master.livenessProbe" (dict "port" $probePort) | indent 10 }}
+          readinessProbe:
+            tcpSocket:
+              port: {{ $probePort }}
+            initialDelaySeconds: {{ .Values.master.readinessProbe.initialDelaySeconds }}
```

Comment:
Need a null check like `{{- if .Values.master.readinessProbe }}`?

Reply:
If there are defaults, then why do you need a null check?

Reply:
makes sense